iocore/hostdb/P_RefCountCache.h

```diff
@@ -339,7 +339,7 @@ RefCountCachePartition<C>::count() const
 
 template <class C>
 void
-RefCountCachePartition<C>::copy(Vec<RefCountCacheHashEntry *> &items)
+RefCountCachePartition<C>::copy(std::vector<RefCountCacheHashEntry *> &items)
```

Comment:
`const&`

Reply:
I do not get where do you indent to add const& qualifier here? Into method's declaration/definition? If so it causes compilation error.

Reply:
What he mean I believe is to change 

`RefCountCachePartition<C>::copy(std::vector<RefCountCacheHashEntry *> &items)`

to

`RefCountCachePartition<C>::copy(std::vector<RefCountCacheHashEntry *> const &items) `

Reply:
@dragon512 FYI:

In file included from /home/istar/git_repos/trafficserver-fork/iocore/hostdb/I_HostDBProcessor.h:34:0,
                 from /home/istar/git_repos/trafficserver-fork/iocore/dns/SRV.h:29,
                 from /home/istar/git_repos/trafficserver-fork/iocore/dns/I_DNSProcessor.h:27,
                 from /home/istar/git_repos/trafficserver-fork/iocore/dns/P_DNSConnection.h:35,
                 from P_UnixNet.h:111,
                 from P_Net.h:97,
                 from BIO_fastopen.cc:24:
/home/istar/git_repos/trafficserver-fork/iocore/hostdb/P_RefCountCache.h: In member function 'void RefCountCachePartition<C>::copy(const std::vector<RefCountCacheHashEntry*>&)':
/home/istar/git_repos/trafficserver-fork/iocore/hostdb/P_RefCountCache.h:346:24: error: no matching function for call to 'std::vector<RefCountCacheHashEntry*>::push_back(RefCountCacheHashEntry*&) const'
     items.push_back(val);
                        ^
In file included from /usr/include/c++/5/vector:64:0,
                 from /usr/include/c++/5/bits/random.h:34,
                 from /usr/include/c++/5/random:49,
                 from /usr/include/c++/5/bits/stl_algo.h:66,
                 from /usr/include/c++/5/algorithm:62,
                 from ../../lib/ts/string_view.h:32,
                 from ../../lib/ts/ink_memory.h:32,
                 from ../../lib/ts/defalloc.h:26,
                 from ../../lib/ts/List.h:61,
                 from /home/istar/git_repos/trafficserver-fork/iocore/eventsystem/I_Continuation.h:40,
                 from /home/istar/git_repos/trafficserver-fork/iocore/eventsystem/I_Action.h:30,
                 from /home/istar/git_repos/trafficserver-fork/iocore/eventsystem/I_EventSystem.h:31,
                 from /home/istar/git_repos/trafficserver-fork/iocore/eventsystem/P_EventSystem.h:36,
                 from P_Net.h:94,
                 from BIO_fastopen.cc:24:
/usr/include/c++/5/bits/stl_vector.h:913:7: note: candidate: void std::vector<_Tp, _Alloc>::push_back(const value_type&) [with _Tp = RefCountCacheHashEntry*; _Alloc = std::allocator<RefCountCacheHashEntry*>; std::vector<_Tp, _Alloc>::value_type = RefCountCacheHashEntry*] <near match>
       push_back(const value_type& __x)
       ^
/usr/include/c++/5/bits/stl_vector.h:913:7: note:   passing 'const std::vector<RefCountCacheHashEntry*>*' as 'this' argument discards qualifiers
/usr/include/c++/5/bits/stl_vector.h:931:7: note: candidate: void std::vector<_Tp, _Alloc>::push_back(std::vector<_Tp, _Alloc>::value_type&&) [with _Tp = RefCountCacheHashEntry*; _Alloc = std::allocator<RefCountCacheHashEntry*>; std::vector<_Tp, _Alloc>::value_type = RefCountCacheHashEntry*] <near match>
       push_back(value_type&& __x)
       ^
/usr/include/c++/5/bits/stl_vector.h:931:7: note:   conversion of argument 1 would be ill-formed:
At global scope:
cc1plus: warning: unrecognized command line option '-Wno-noexcept-type'
cc1plus: warning: unrecognized command line option '-Wno-format-truncation'
Makefile:1013: recipe for target 'BIO_fastopen.o' failed

Reply:
Ya I see this is a copyTo, not a copy from... so const is wrong.